src/graph.c

```diff
@@ -42,6 +46,9 @@ RAI_Graph *RAI_GraphCreate(RAI_Backend backend, RAI_Device device,
   if (backend == RAI_BACKEND_TENSORFLOW) {
     return RAI_GraphCreateTF(backend, device, graphdef, graphlen);
   }
+  else if (backend == RAI_BACKEND_TORCH) {
+    return RAI_GraphCreateTorch(backend, device, graphdef, graphlen);
+  }
 
   return NULL;
```

Comment:
could this happened? if not assert?

Reply:
I get the previous comment now: Torch graphs are fully supported. See in `pytorch-minimal.py` and `load_torch_model.sh` in `examples/models` for an example on how to trace+export a graph from PyTorch and run it in Redis. See here for reference: https://pytorch.org/tutorials/advanced/cpp_export.html

Reply:
I actually meant the return NULL if we reach there its a bug right? So it might be better to put an assert there so we will now it happened.

Reply:
Oh right, sorry. Yes, great point.

Reply:
Done